src/cpp/core/DDSRouter.cpp

```diff
@@ -87,23 +91,73 @@ ReturnCode DDSRouter::reload_configuration(
 }
 
 ReturnCode DDSRouter::start() noexcept
+{
+    ReturnCode ret = start_();
+    if (ret == ReturnCode::RETCODE_OK)
+    {
+        logUser(DDSROUTER, "Starting DDSRouter");
```

Comment:
In general, I'd prefer DDS Router separated in log traces unless it refers to the actual `DDSRouter` class
```suggestion
        logUser(DDSROUTER, "Starting DDS Router");
```

Reply:
:+1: